src/modules/auth/controllers/het-archief.controller.ts

```diff
@@ -134,11 +134,13 @@ export class HetArchiefController {
 			// determine user group
 			const userGroup = await this.idpService.determineUserGroup(ldapUser);
 
+			const apps = get(ldapUser, 'attributes.apps', []);
```

Comment:
avoid using lodash get, since it has poor type checking.
```suggestion
			const apps = ldapUser?.attributes?.apps ?? [];
```